sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_operations.py

```diff
@@ -790,29 +784,29 @@ def _release_cloud_events(
     @api_version_validation(
         params_added_on={"2023-10-01-preview": ["release_delay_in_seconds"]},
     )
-    def _release_cloud_events(
+    def _release(
         self,
         topic_name: str,
         event_subscription_name: str,
         release_options: Union[_models._models.ReleaseOptions, JSON, IO[bytes]],
         *,
-        release_delay_in_seconds: Optional[Union[int, _models.ReleaseDelay]] = None,
+        release_delay_in_seconds: Optional[Union[str, _models.ReleaseDelay]] = None,
         **kwargs: Any
     ) -> _models.ReleaseResult:
-        """Release batch of Cloud Events. The server responds with an HTTP 200 status code if the request
-        is successfully accepted. The response body will include the set of successfully released
-        lockTokens, along with other failed lockTokens with their corresponding error information.
+        """Release a batch of Cloud Events. The response will include the set of successfully released
+        lock tokens, along with other failed lock tokens with their corresponding error information.
+        Successfully released events can be received by consumers.
 
         :param topic_name: Topic Name. Required.
         :type topic_name: str
         :param event_subscription_name: Event Subscription Name. Required.
         :type event_subscription_name: str
         :param release_options: ReleaseOptions. Is one of the following types: ReleaseOptions, JSON,
          IO[bytes] Required.
-        :type release_options: ~azure.eventgrid.models.ReleaseOptions or JSON or IO[bytes]
+        :type release_options: ~azure.eventgrid.models._models.ReleaseOptions or JSON or IO[bytes]
         :keyword release_delay_in_seconds: Release cloud events with the specified delay in seconds.
-         Known values are: 0, 10, 60, 600, and 3600. Default value is None.
-        :paramtype release_delay_in_seconds: int or ~azure.eventgrid.models.ReleaseDelay
+         Known values are: "0", "10", "60", "600", and "3600". Default value is None.
```

Comment:
We might want to add some wording that None means "release right away" or something to be clear

Reply:
gen code - will add to my patch :)